storage/block/how-to/delete-a-volume.mdx

```diff
@@ -21,22 +21,19 @@ categories:
   - You have [created](/storage/block/how-to/create-a-volume/) a Block Storage volume
 </Message>
 
-<Message type="note">
-  The only way for you to delete a Block volume, is if it is detached from the Instance. Refer to our documentation to find out [how to detach a Block volume from an Instance](/storage/block/how-to/detach-a-volume/).
+<Message type="important">
+  - You must detach your volume from the Instance it is linked to. Refer to our documenation to find out [how to do so](/storage/block/how-to/detach-a-volume/).
+  - [Unmounting](/storage/block/api-cli/unmounting-a-volume/) your device from the operating system before detaching the Block Storage volume, is highly recommended, to avoid data loss.
```

Comment:
```suggestion
  - We highly recommend that you [unmount](/storage/block/api-cli/unmounting-a-volume/) your device from the operating system before detaching the Block Storage volume, to avoid data loss.
```